response_operations_ui/views/errors.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from flask import Blueprint, render_template
+from structlog import wrap_logger
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+error_bp = Blueprint('error_bp', __name__, template_folder='templates/errors')
+
+
+@error_bp.route('/500', methods=['GET', 'POST'])
```

Comment:
I guess this only needs to be GET, it's unlikely something will post to this page?